user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -19,111 +19,295 @@ redirect_from:
-simple and asks very few questions. (It's actually easier to install Qubes right
-now than most other Linux distributions!)
+<img src="/attachment/wiki/InstallationGuide/add-ons.png">
 
-The installer loads Xen right at the beginning, so chances are high that if you
-can see the installer's graphical screen and you pass the compatibility check that
-runs immediately after that, Qubes will work on your system. :)
+Go instead to the Software selection tab, where you can choose which software to install alongside Qubes OS. Two Add-Ons are available :
 
-If you're an advanced user, and you'd like to customize your installation, please see [Custom Installation].
+* **Debian 9 (stretch) template** : Install these templates if you wish to base some of your Qubes virtual machines on Debian instead of Fedora. 
+* **Whonix** : Install Whonix templates if you wish some of your qubes to be based on Whonix. Whonix let you route your entire network traffic trough Tor if you see fit. For more information about Whonix, have a look at their [website](https://www.whonix.org/). Depending on your threat model, you may need to install Whonix templates right away.
 
+Note that you will also be able to install  Add-Ons after the installation is completed. If you wish for your system to be more lightweight, 
+do not hesitate to un-check those options. 
 
-Installing to a USB drive
--------------------------
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b> Note : </b> By default, Qubes OS comes preinstalled with the lightweight Xfce desktop environnement for dom0, the main domain. Other desktop environments will be available to you after the installation is completed, although they may not be officially supported. 
+</div>
 
-Installing an operating system onto a USB drive can be a convenient and secure
-method of ensuring that your data is protected. Be advised that a minimum
-storage of 32 GB is required on the USB drive. This installation process may
-take longer than an installation on a standard hard disk. The installation
-process is identical to using a hard disk in conjunction with two exceptions:
+Click on "Done" as soon as you have made your choice to go back to Installation summary screen. 
 
-* Select the USB as the storage location for the OS. 
+<h3>Installation destination</h3>
 
-* Leave the option checked to “Automatically configure my Qubes installation to
-the disk(s) I selected and return me to the main menu”.
+Under the System section, you need to pick the installation destination. For this step to be completed, you need to select which storage device you would like your system to be installed on. By default, only local storage devices are shown. Under the Device Selection section, make sure that you select the correct installation destination. Ensure that your your target destination has a least 32 GiB of free space available.   
```

Comment:
Not sure what you mean by "By default, only local storage devices are shown"? We don't support installation on network disks at all.

Reply:
I deleted this phrase